template/src/locale/I18nConfig.ts

```diff
@@ -1,28 +1,34 @@
-import I18n from 'i18n-js'
+import i18n from 'i18next'
+import {getI18n, initReactI18next} from 'react-i18next'
```

Comment:
Why do we need to change our `i18n` feature, I think we can move this into a different PR

Reply:
updated